### ISSUE TITLE
BoxLoader compatibility

### ISSUE THREAD

**Issue by irvnriir** [zen-mod/ZEN#524]
**Arma 3 Version:** `2.00` (stable / rc / dev)
**CBA Version:** `last` (stable / dev + commit hash)
**ZEN Version:** `last` (stable / dev + commit hash)

**Mods:**
```
- CBA_A3
- ZEN
- BoxLoader
```

**Description:**
If vehicles is Deep Copy- or Export Mission SQF- loaded, after deleting them by Zeus, this mod's system Objects stay .

**Where did the issue occur?**
- Self-Hosted Multiplayer

**Comment by mharis001** [comment]
I think the best we can do for this situation is to provide an API that allows objects to be blacklisted from the serialization functions (instead of direct compatibility for a specific mod).

**Comment by CreepPork** [comment]
Yeah, I don't think it is up to ZEN to add compatibility to other, non-trivial mods. Providing an API other mods could tap into would be nice.

**Comment by irvnriir** [comment]
@CreepPork I posted it in case that related to inconsistency inside ZEN, its about what ZEN do with other mods and not what they do .
